tests/vmi_lifecycle_test.go

```diff
@@ -259,11 +259,14 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 				Should(ContainSubstring(`"subcomponent":"libvirt"`))
 		})
 
-		DescribeTable("log libvirtd debug logs should be", func(vmiLabels, vmiAnnotations map[string]string, expectDebugLogs bool) {
+		DescribeTable("log libvirtd debug logs should be", func(vmiLabelKey string, vmiLabelValue string, vmiAnnotationKey string, vmiAnnotationValue string, expectDebugLogs bool) {
```

Comment:
This is shorter:
`vmiLabelKey, vmiLabelValue, vmiAnnotationKey, vmiAnnotationValue string, expectDebugLogs bool`

Reply:
Done